app/graph/types/project_media_type.rb

```diff
@@ -0,0 +1,77 @@
+ProjectMediaType = GraphqlCrudOperations.define_default_type do
+  name 'ProjectMedia'
+  description 'ProjectMedia type'
+
+  interfaces [NodeIdentification.interface]
+
+  field :id, field: GraphQL::Relay::GlobalIdField.new('ProjectMedia')
+  field :updated_at, types.String
+  field :created_at, types.String
+  field :media_id, types.Int
+  field :project_id, types.Int
+  field :user_id, types.Int
+  field :permissions, types.String
+  field :project do
+    type -> { ProjectType }
+
+    resolve -> (project_media, _args, _ctx) {
```

Comment:
OK cool. I thought it had to do with our own "context" which we're trying to remove... A good reminder that choosing names that are too generic can lead to confusion.